src/avif.c

```diff
@@ -400,8 +400,8 @@ static clapFraction calcCenter(int32_t dim)
     clapFraction f;
     f.n = dim >> 1;
     f.d = 1;
-    if ((dim % 2) == 1) {
-        f.n = (f.n * 2) + 1;
+    if ((dim % 2) != 0) {
```

Comment:
The change from `(dim % 2) == 1` to `(dim % 2) != 0` is to handle the unlikely case that we pass a negative `dim` argument to this function.